src/ScottPlot4/ScottPlot.Avalonia/AvaPlot.axaml.cs

```diff
@@ -75,6 +75,11 @@ public partial class AvaPlot : UserControl
         [Obsolete("Reference Plot instead of plt")]
         public ScottPlot.Plot plt => Plot;
 
+        static AvaPlot()
```

Comment:
I'm not quite familiar, but do all `ContextMenu`s forward their events to this instance?

In any case, I don't think this (or the existing subscription) will fire if someone swaps out the `ContextMenu` instance rather than mutate it.

e.g. `avaPlot.ContextMenu = foo` rather than `avaPlot.ContextMenu.Items = foo`

This is probably incorrect, and I think we might to add some logic to `AvaPlot_PropertyChanged` to handle that case. In principle we could add a handler for `this.ContextMenu` whenever it changes, but I do like how you used the static constructor for it instead.

Reply:
That was a convoluted comment, so I think I'll summarize:
- I don't think `avaPlot.ContextMenu = foo` will fire the event
- I think this can be solved by modifying `AvaPlot_PropertyChanged`, but I'm open to more elegant approaches